### PR TITLE
EN 6803 close arwen after genesis

### PR DESCRIPTION
**Pull request by iulianpascalau**
- closed vm container to close the arwen process
- switched to new logger version
- main closing wait will be done only if the components could not be gracefully closed

### CODE REVIEW

File: cmd/node/main.go [hunk 1]
```diff
@@ -1332,10 +1332,17 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
...
 	go func() {
-		closeAllComponents(log, dataComponents, triesComponents, networkComponents)
+		closeAllComponents(log, dataComponents, triesComponents, networkComponents, chanCloseComponents)
 	}()
-	time.Sleep(maxTimeToClose)
+
+	select {
+	case <-chanCloseComponents:
+	case <-time.After(maxTimeToClose):
+		log.Debug("closeAllComponents did not finished on time, force closing the node")
```
Comment (sasurobert) [inline cmd/node/main.go:1343]: log.Warn / error maybe ?

Comment (sasurobert) [reply cmd/node/main.go:1343]: add error in the message in order to find it easily.

File: go.sum [hunk 2]
```diff
@@ -12,6 +12,8 @@ github.com/ElrondNetwork/concurrent-map v0.1.2/go.mod h1:3XwSwn4JHI0lrKxWLZvtp53
 github.com/ElrondNetwork/elrond-go-logger v1.0.2/go.mod h1:e5D+c97lKUfFdAzFX7rrI2Igl/z4Y0RkKYKWyzprTGk=
 github.com/ElrondNetwork/elrond-go-logger v1.0.3 h1:PHH6m0vhgBbChd8Cu3Oda5LS1FtPcQRQw8DGM5osZ3Y=
 github.com/ElrondNetwork/elrond-go-logger v1.0.3/go.mod h1:e5D+c97lKUfFdAzFX7rrI2Igl/z4Y0RkKYKWyzprTGk=
+github.com/ElrondNetwork/elrond-go-logger v1.0.4 h1:i5Yu4qyjTZDwvBY/ykbNpp2SP9jxwk/QTivRwSZSTAQ=
```
Comment (sasurobert) [inline go.sum:15]: go mod tidy

Comment (iulianpascalau) [reply go.sum:15]: done

File: cmd/node/main.go (hunk 1 above)
Comment (iulianpascalau) [reply cmd/node/main.go]: done

**Pull request merged**